static/app/views/settings/organization/navigationConfiguration.tsx

```diff
@@ -114,12 +114,8 @@ const organizationNavigation: NavigationSection[] = [
         path: `${organizationSettingsPathPrefix}/dynamic-sampling/`,
         title: t('Dynamic Sampling'),
         description: t('Manage your sampling rate'),
-        show: ({organization, isSelfHosted}) =>
-          !!(
-            organization &&
-            hasDynamicSamplingCustomFeature(organization) &&
-            isSelfHosted
-          ),
+        show: ({organization}) =>
+          !!organization && hasDynamicSamplingCustomFeature(organization),
```

Comment:
Just as a sanity check: Is the boolean cast purposely not on the entire expression, but just the organization?

Reply:
only needed on organization, `hasDynamicSamplingCustomFeature`.
TS would complain otherwise.